### ISSUE TITLE
Err when deleting alerts/Unable to delete alerts

### ISSUE THREAD

**Issue by pevma** [StamusNetworks/scirius#100]
With Scirius 1.2 - when a selected sid/alert is needed to be deleted - aka "delete generated alerts" it results in an error:

```

Environment:


Request Method: POST
Request URL: https://selks/rules/rule/2018789/delete

Django Version: 1.8.17
Python Version: 2.7.9
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django_tables2',
 'bootstrap3',
 'dbbackup',
 'rules',
 'suricata',
 'accounts',
 'revproxy')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'scirius.loginrequired.LoginRequiredMiddleware',
 'scirius.utils.TimezoneMiddleware')


Template error:
In template /usr/share/python/scirius/lib/python2.7/site-packages/rules/templates/rules/delete_alerts.html, error at line 52
   Parameter "form" should contain a valid Django Form.

   42 : </svg>



   43 : </div> <!-- timeline -->



   44 : </div> <!-- col -->



   45 : </div> <!-- row -->



   46 : 



   47 : <div class="row">



   48 : <h2 class="title">Are you sure you want to delete all stored alerts for {{ object.pk }}?</h2>



   49 : 



   50 : <form class="form" action="{% url 'delete_alerts' object.pk %}" method="post">{% csrf_token %}



   51 : 



   52 :      {% bootstrap_form comment_form %} 



   53 : <button class="btn btn-primary" type="submit">



   54 : <span class="glyphicon glyphicon-trash"> Delete alerts</span>



   55 : </button>



   56 : </form>



   57 : </div> <!-- row -->



   58 : 



   59 : </div>



   60 : 



   61 : <script>



   62 : {% if probes %}


Traceback:
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  132.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/share/python/scirius/lib/python2.7/site-packages/rules/views.py" in delete_alerts
  560.                     return scirius_render(request, 'rules/delete_alerts.html', context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/scirius/utils.py" in scirius_render
  104.     return render(request, template, context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/shortcuts.py" in render
  67.             template_name, context, request=request, using=using)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/loader.py" in render_to_string
  99.         return template.render(context, request)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/backends/django.py" in render
  74.         return self.template.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in render
  210.                     return self._render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/loader_tags.py" in render
  135.         return compiled_parent._render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in _render
  202.         return self.nodelist.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/loader_tags.py" in render
  65.                 result = block.nodelist.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in render
  905.                 bit = self.render_node(node, context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/debug.py" in render_node
  79.             return node.render(context)
File "/usr/share/python/scirius/lib/python2.7/site-packages/django/template/base.py" in render
  1197.                     return func(*resolved_args, **resolved_kwargs)
File "/usr/share/python/scirius/lib/python2.7/site-packages/bootstrap3/templatetags/bootstrap3.py" in bootstrap_form
  330.     return render_form(*args, **kwargs)
File "/usr/share/python/scirius/lib/python2.7/site-packages/bootstrap3/forms.py" in render_form
  45.     return renderer_cls(form, **kwargs).render()
File "/usr/share/python/scirius/lib/python2.7/site-packages/bootstrap3/renderers.py" in __init__
  156.                 'Parameter "form" should contain a valid Django Form.')

Exception Type: BootstrapError at /rules/rule/2018789/delete
Exception Value: Parameter "form" should contain a valid Django Form.
```

**Comment by pevma** [comment]
To clarify -  this is after and upgrade/migration from Scirius 1.11 to 1.2

**Comment by regit** [comment]
Were your session on scirius with a non privileged user ?

**Comment by pevma** [comment]
As root on SELKS

**Comment by regit** [comment]
You've got an error 500 because you enter into some buggy error code. The problem is because you are using ES 5 but this is not declared correctly in local_settings. You need to remove `ELASTICSEARCH_2X=True` from local_settings and replace it with `ELASTICSEARCH_VERSION=5`

I'm gonna push a bugfix sor the error 500 but your settings needs to be fixed.

**Comment by regit** [comment]
Fixed by 2b61f4a5dea15411800a225a6ef6bf5d1254b003
